docs/workflow/pycbc_make_coinc_search_workflow.rst

```diff
@@ -608,53 +608,250 @@ If you have a subdax that failed, ``pegasus_analyzer`` will provide you with a c
+    116912/H1-INSPIRAL_FULL_DATA_JOB1-1169120586-1662.hdf file://localhost/home/dbrown/projects/aligo/o2/analysis-4/o2-analysis-4/output/full_data/H1-INSPIRAL_FULL_DATA_JOB1-1169120586-1662.hdf pool="local"
+
+In this case, Pegasus will delete the jobs from the workflow that create the files ``H1-VETOTIME_CAT3-1169107218-1066800.xml``, ``L1-VETOTIME_CAT3-1169107218-1066800.xml``, ``116912/H1-INSPIRAL_FULL_DATA_JOB0-1169120586-1662.hdf``, and ``116912/H1-INSPIRAL_FULL_DATA_JOB1-1169120586-1662.hdf`` when it plans the workflow. Insted, the data will be re-used from the URLs specified in the cache. Since ``site="local"`` for these files, Pegasus expects that the files all exist on the host where the workflow is run from.
+
+To re-use data from a remote cluster, the URLs must contain a file transfer
+mechanism and the ``SITE`` should be set to ``remote``. For example, if the
+files listed in the example above are available on
+``sugwg-condor.phy.syr.edu`` and you want to re-use them in a workflow on the
+AEI Atlas cluster, then the cache file would contain::
+
+    H1-VETOTIME_CAT3-1169107218-1066800.xml gsiftp://sugwg-condor.phy.syr.edu/home/dbrown/projects/aligo/o2/analysis-4/o2-analysis-4/output/results/1._analysis_time/1.01_segment_data/H1-VETOTIME_CAT3-1169107218-1066800.xml pool="remote"
+    L1-VETOTIME_CAT3-1169107218-1066800.xml gsiftp://sugwg-condor.phy.syr.edu/home/dbrown/projects/aligo/o2/analysis-4/o2-analysis-4/output/results/1._analysis_time/1.01_segment_data/L1-VETOTIME_CAT3-1169107218-1066800.xml pool="remote"
+    116912/H1-INSPIRAL_FULL_DATA_JOB0-1169120586-1662.hdf gsiftp://sugwg-condor.phy.syr.edu/home/dbrown/projects/aligo/o2/analysis-4/o2-analysis-4/output/full_data/H1-INSPIRAL_FULL_DATA_JOB0-1169120586-1662.hdf pool="remote"
+    116912/H1-INSPIRAL_FULL_DATA_JOB1-1169120586-1662.hdf gsiftp://sugwg-condor.phy.syr.edu/home/dbrown/projects/aligo/o2/analysis-4/o2-analysis-4/output/full_data/H1-INSPIRAL_FULL_DATA_JOB1-1169120586-1662.hdf pool="remote"
+
+Note that the URL now contains ``gsiftp://sugwg-condor.phy.syr.edu`` rather
+than ``file://localhost`` and the files are listes as ``pool="remote"`` rather
+than ``pool="local"``. Pegasus will re-use these data files adding in
+file transfer jobs to the workflow to move them into the appropriate
+locations.
+
+Once a cache file has been constructed, to enable data re-use, you follow the
+standard instructions for planning and submitting the workflow in the section
+:ref:`coincworkflowplan`, but add the ``--cache-file`` argument that points to
+the cache file that you have created. For example:: 
+
+    pycbc_submit_dax --accounting-group ligo.dev.o1.cbc.explore.test --dax s6d_chunk3.dax --cache-fole /path/to/prior_data.map
+
+will use the URLs from the file ``/path/to/prior_data.map`` to implement
+data re-use and subsequent workflow reduction.
+
+The main key to data re-use is generating the cache file. There is no rule for
```

Comment:
suggest to rephrase : 'Which file URLs should be included in the reuse cache?  There is no single correct way of deciding this ...'